packages/unlock-js/src/walletService.ts

```diff
@@ -63,7 +65,7 @@ export default class WalletService extends UnlockService {
    */
   // eslint-disable-next-line no-underscore-dangle
   // TODO: Do we need this???
-  async _handleMethodCall(methodCall) {
+  async _handleMethodCall(methodCall: any) {
```

Comment:
this is a Promise, in the line below we are using await on `methodCall`
```suggestion
  async _handleMethodCall(methodCall: Promise<any>) {
```

Reply:
Can also be a non promise. The distinction doesn't matter when you use any.